src/Importers/DBImport.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace TheRealMVP\Importers;
+
+class DBImport
+{
+    protected \PDO $pdoConnection;
+    protected GetAPI $curlConnection;
```

Comment:
Final comment!!!!!

The name of this property is not very semantic, it sounds like it would just be a curl connection but actually its your API object.

Suggest renaming to something like APIConnection instead, just so it's more semantic!